deployment/terraform/assets/cluster.tf

```diff
@@ -108,6 +110,8 @@ resource "aws_instance" "metrics_server" {
   provisioner "remote-exec" {
     inline = [
       "while [ ! -f /var/lib/cloud/instance/boot-finished ]; do echo 'Waiting for cloud-init...'; sleep 1; done",
+      "while ! netcat -z -v security.ubuntu.com 80; do echo 'Waiting for connectivity to security.ubuntu.com...'; sleep 1; done",
+      "while ! netcat -z -v us-east-1.ec2.archive.ubuntu.com 80; do echo 'Waiting for connectivity to us-east-1.ec2.archive.ubuntu.com...'; sleep 1; done",
```

Comment:
Should we consider importing these from a shared script file? It seems like a lot of duplication is happening here.